solvers/H3LAPD/ParticleSystems/NeutralParticleSystem.hpp

```diff
@@ -250,12 +251,12 @@ class NeutralParticleSystem {
     std::vector<int> components = {0};
     m_field_project->project(syms, components);
     if (m_low_order_project) {
-      FieldUtils::Interpolator interpolator{};
+      FU::InterpolatorSharedPtr interpolator;
```

Comment:
Does this need a make_shared to actually call the constructor?

Reply:
Yep
Think I'll change it to what you have in #234 anyway - no real need for a shared pointer here